.github/workflows/build-push-docker-image.yml

```diff
@@ -0,0 +1,50 @@
+name: Build the Docker container images for Stepup Webauthn
+
+on:
+  push:
+    branches:
+      - main
+      - prod-docker-containers
+
+jobs:
+  build-push-docker-image:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push the Production image
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          file: docker/Dockerfile.prod
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: |
+            ghcr.io/openconext/stepup-webauthn/stepup-webauthn:prod
+            ghcr.io/openconext/stepup-webauthn/stepup-webauthn:${{ github.sha }}
+
+      - name: Build and push the Development image
```

Comment:
For discussion: If all php dev images are the same (apache + php + xdebug) we might publish one container to rule them all in the basecontainers project

Reply:
Agreed! I like this proposal that we also discussed in today's meeting. I suggest we take a look at the end, if all the applications are very similar, this might be very useful.